From 38a8c496f9de4331e6dc104758fb4c27f27eb32b Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Wed, 17 Apr 2019 19:43:09 +0200
Subject: [PATCH] board: rename tqma8qxs -> tqma8xxs

This rename should (hopefully) correct the naming mess:

module is TQMa8XxS (X for the i.MX8 X family, x for CPU variant)

therefore:

- variant with i.MXDX is named TQMa8XDS
- variant with i.MXQXP is named TQMa8XQPS

We rename / move here:

- config names
- file names
- board names
- Kconfig symbols

While at it, improve naming of baseboard Kconfig choice names

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/dts/Makefile                         |  4 +-
 ...dts => fsl-imx8dx-tqma8xds-mb-smarc-2.dts} |  2 +-
 ...s => fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts} |  6 +-
 ...a8qxps.dtsi => fsl-imx8qxp-tqma8xqps.dtsi} |  4 +-
 arch/arm/mach-imx/imx8/Kconfig                |  6 +-
 board/tqc/tqma8qxs/Kconfig                    | 95 -------------------
 board/tqc/tqma8qxs/Makefile                   |  9 --
 board/tqc/tqma8xxs/Kconfig                    | 95 +++++++++++++++++++
 board/tqc/tqma8xxs/Makefile                   |  9 ++
 .../tqma8xxs-mb-smarc-2.c}                    |  0
 .../tqma8qxs.c => tqma8xxs/tqma8xxs.c}        |  0
 ...nfig => tqma8xds_mb_smarc_2_mmc_defconfig} |  4 +-
 ...fig => tqma8xqps_mb_smarc_2_mmc_defconfig} |  2 +-
 ...qxs-mb-smarc-2.h => tqma8xxs-mb-smarc-2.h} |  8 +-
 include/configs/{tqma8qxs.h => tqma8xxs.h}    | 22 ++---
 15 files changed, 133 insertions(+), 133 deletions(-)
 rename arch/arm/dts/{fsl-imx8dx-tqma8dxs-mb-smarc-2.dts => fsl-imx8dx-tqma8xds-mb-smarc-2.dts} (94%)
 rename arch/arm/dts/{fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts => fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts} (96%)
 rename arch/arm/dts/{fsl-imx8qxp-tqma8qxps.dtsi => fsl-imx8qxp-tqma8xqps.dtsi} (98%)
 delete mode 100644 board/tqc/tqma8qxs/Kconfig
 delete mode 100644 board/tqc/tqma8qxs/Makefile
 create mode 100644 board/tqc/tqma8xxs/Kconfig
 create mode 100644 board/tqc/tqma8xxs/Makefile
 rename board/tqc/{tqma8qxs/tqma8qxs-mb-smarc-2.c => tqma8xxs/tqma8xxs-mb-smarc-2.c} (100%)
 rename board/tqc/{tqma8qxs/tqma8qxs.c => tqma8xxs/tqma8xxs.c} (100%)
 rename configs/{tqma8dxs_mb_smarc_2_mmc_defconfig => tqma8xds_mb_smarc_2_mmc_defconfig} (96%)
 rename configs/{tqma8qxs_mb_smarc_2_mmc_defconfig => tqma8xqps_mb_smarc_2_mmc_defconfig} (98%)
 rename include/configs/{tqma8qxs-mb-smarc-2.h => tqma8xxs-mb-smarc-2.h} (76%)
 rename include/configs/{tqma8qxs.h => tqma8xxs.h} (97%)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index e4bff0806e..d83e92578c 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -473,7 +473,7 @@ dtb-$(CONFIG_ARCH_IMX8M) += fsl-imx8mq-evk.dtb \
 
 dtb-$(CONFIG_ARCH_IMX8) += fsl-imx8dx-ddr3-arm2.dtb \
 		fsl-imx8dx-tqma8xd-mba8xx.dtb \
-		fsl-imx8dx-tqma8dxs-mb-smarc-2.dtb \
+		fsl-imx8dx-tqma8xds-mb-smarc-2.dtb \
 		fsl-imx8qm-ddr4-arm2.dtb \
 		fsl-imx8qm-lpddr4-arm2.dtb \
 		fsl-imx8qm-mek.dtb \
@@ -481,7 +481,7 @@ dtb-$(CONFIG_ARCH_IMX8) += fsl-imx8dx-ddr3-arm2.dtb \
 		fsl-imx8qxp-lpddr4-arm2.dtb \
 		fsl-imx8qxp-mek.dtb \
 		fsl-imx8qxp-tqma8xqp-mba8xx.dtb \
-		fsl-imx8qxp-tqma8qxps-mb-smarc-2.dtb
+		fsl-imx8qxp-tqma8xqps-mb-smarc-2.dtb
 
 dtb-$(CONFIG_RCAR_GEN3) += \
 	r8a7795-h3ulcb.dtb \
diff --git a/arch/arm/dts/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts b/arch/arm/dts/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
similarity index 94%
rename from arch/arm/dts/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts
rename to arch/arm/dts/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
index 49621b8b14..c5cbdd3f09 100644
--- a/arch/arm/dts/fsl-imx8dx-tqma8dxs-mb-smarc-2.dts
+++ b/arch/arm/dts/fsl-imx8dx-tqma8xds-mb-smarc-2.dts
@@ -12,7 +12,7 @@
  * GNU General Public License for more details.
  */
 
-#include "fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts"
+#include "fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts"
 
 / {
 	model = "TQ Systems i.MX8DX TQMa8XDS on MB-SMARC-2";
diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts b/arch/arm/dts/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
similarity index 96%
rename from arch/arm/dts/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts
rename to arch/arm/dts/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
index 98f3d75b13..85633f44e1 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8qxps-mb-smarc-2.dts
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqps-mb-smarc-2.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 TQ Systems GmbH
+ * Copyright 2018 - 2019 TQ Systems GmbH
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -15,11 +15,11 @@
 /dts-v1/;
 
 #include <dt-bindings/net/ti-dp83867.h>
-#include "fsl-imx8qxp-tqma8qxps.dtsi"
+#include "fsl-imx8qxp-tqma8xqps.dtsi"
 
 / {
 	model = "TQ Systems i.MX8QXP TQMa8xXS on MB-SMARC-2";
-	compatible = "tq,tqma8qxps-mb-smarc-2", "tq,tqma8qxps", "fsl,imx8qxp";
+	compatible = "tq,tqma8xqps-mb-smarc-2", "tq,tqma8xqps", "fsl,imx8qxp";
 
 	chosen {
 		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x5a060000,115200";
diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8qxps.dtsi b/arch/arm/dts/fsl-imx8qxp-tqma8xqps.dtsi
similarity index 98%
rename from arch/arm/dts/fsl-imx8qxp-tqma8qxps.dtsi
rename to arch/arm/dts/fsl-imx8qxp-tqma8xqps.dtsi
index eb1a6b2b52..604f5bae43 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8qxps.dtsi
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqps.dtsi
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 TQ Systems GmbH
+ * Copyright 2018 - 2019 TQ Systems GmbH
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -20,7 +20,7 @@
 
 / {
 	model = "TQ Systems i.MX8QXP TQMa8xXS";
-	compatible = "tq,tqma8qxps", "fsl,imx8qxp";
+	compatible = "tq,tqma8xqps", "fsl,imx8qxp";
 
 	regulators {
 		compatible = "simple-bus";
diff --git a/arch/arm/mach-imx/imx8/Kconfig b/arch/arm/mach-imx/imx8/Kconfig
index 045397e9e2..46f685d93b 100644
--- a/arch/arm/mach-imx/imx8/Kconfig
+++ b/arch/arm/mach-imx/imx8/Kconfig
@@ -105,8 +105,8 @@ config TARGET_TQMA8XX
 	select SPI_FLASH_SST
 	select SPI_FLASH_WINBOND
 
-config TARGET_TQMA8QXS
-	bool "Support TQMa8xXS board"
+config TARGET_TQMA8XXS
+	bool "Support TQMa8XxS board"
 	select BOARD_LATE_INIT
 	select IMX8QXP
 	select ARCH_MISC_INIT
@@ -141,6 +141,6 @@ source "board/freescale/imx8qm_mek/Kconfig"
 source "board/freescale/imx8qxp_arm2/Kconfig"
 source "board/freescale/imx8qxp_mek/Kconfig"
 source "board/tqc/tqma8xx/Kconfig"
-source "board/tqc/tqma8qxs/Kconfig"
+source "board/tqc/tqma8xxs/Kconfig"
 
 endif
diff --git a/board/tqc/tqma8qxs/Kconfig b/board/tqc/tqma8qxs/Kconfig
deleted file mode 100644
index 4b9e5be7e5..0000000000
--- a/board/tqc/tqma8qxs/Kconfig
+++ /dev/null
@@ -1,95 +0,0 @@
-if TARGET_TQMA8QXS
-
-config SYS_BOARD
-	default "tqma8qxs"
-
-config SYS_VENDOR
-	default "tqc"
-
-config SYS_CONFIG_NAME
-	default "tqma8qxs"
-
-choice
-	prompt "TQMa8QXS CPU variant"
-	default TQMA8QXS_CPU_MX8QXP
-	help
-	  Select cpu variant for TQMa8QXS
-
-config TQMA8QXS_CPU_MX8QXP
-	bool "TQMa8XQPS with i.MX8QXP"
-	help
-	  Select the CPU variant
-
-config TQMA8QXS_CPU_MX8DX
-	bool "TQMa8XDS with i.MX8DX"
-	help
-	  Select the CPU variant
-
-endchoice
-
-choice
-	prompt "TQMa8XxS RAM size"
-	default TQMA8QXS_RAM_1014MB if TQMA8QX_CPU_MX8QXP
-	default TQMA8QXS_RAM_512MB if TQMA8QXS_CPU_MX8DX
-	help
-	  Select RAM size for TQMa8XxS
-
-config TQMA8QXS_RAM_512MB
-	bool "TQMa8XxS with 512 MiB RAM"
-	help
-	  Select variant with 512 MiB RAM
-
-config TQMA8QXS_RAM_1014MB
-	bool "TQMa8XxS with 1024 MiB RAM"
-	help
-	  Select variant with 1024 MiB RAM
-
-config TQMA8QXS_RAM_2048MB
-	bool "TQMa8XxS with 2048 MiB RAM"
-	help
-	  Select variant with 2048 MiB RAM
-
-endchoice
-
-choice
-	prompt "TQMa8QXS base board variant"
-	default TQMA8QXS_MB_SMARC_2
-	help
-	  Select base board for TQMa8QXS
-
-config TQMA8QXS_MB_SMARC_2
-	bool "TQMa8QXS on MB_SMARC_2 Board"
-	select TQC_SDMMC
-	help
-	  Select the MB_SMARC_2 board. This is a SMARC2 mainboard
-	  etc.
-
-endchoice
-
-config DEFAULT_DEVICE_TREE
-	default "fsl-imx8dx-tqma8dxs-mb-smarc-2" if TQMA8QXS_CPU_MX8DX && TQMA8QXS_MB_SMARC_2
-	default "fsl-imx8qxp-tqma8qxps-mb-smarc-2" if TQMA8QXS_CPU_MX8QXP && TQMA8QXS_MB_SMARC_2
-
-config DEFAULT_FDT_FILE
-	default "fsl-imx8dx-tqma8dxs-mb-smarc-2.dtb" if TQMA8QXS_CPU_MX8DX && TQMA8QXS_MB_SMARC_2
-	default "fsl-imx8qxp-tqma8qxps-mb-smarc-2.dtb" if TQMA8QXS_CPU_MX8QXP && TQMA8QXS_MB_SMARC_2
-
-choice
-	prompt "TQMa8QXS boot device"
-	default TQMA8QXS_BOOT_SDCARD
-	help
-	  Select boot device for TQMa8QXS
-
-config TQMA8QXS_BOOT_SDCARD
-	bool "TQMa8QXS from SD / e-MMC"
-	select SD_BOOT
-	select ENV_IS_IN_MMC
-	help
-	  Select booting from SD / e-MMC
-
-endchoice
-
-
-source "board/tqc/common/Kconfig"
-
-endif
diff --git a/board/tqc/tqma8qxs/Makefile b/board/tqc/tqma8qxs/Makefile
deleted file mode 100644
index bad4cf1ba8..0000000000
--- a/board/tqc/tqma8qxs/Makefile
+++ /dev/null
@@ -1,9 +0,0 @@
-#
-# Copyright 2018 TQ Systems GmbH
-#
-# SPDX-License-Identifier:	GPL-2.0+
-#
-
-obj-y += tqma8qxs.o
-obj-$(CONFIG_TQMA8QXS_MB_SMARC_2) += tqma8qxs-mb-smarc-2.o
-
diff --git a/board/tqc/tqma8xxs/Kconfig b/board/tqc/tqma8xxs/Kconfig
new file mode 100644
index 0000000000..232f3b5d56
--- /dev/null
+++ b/board/tqc/tqma8xxs/Kconfig
@@ -0,0 +1,95 @@
+if TARGET_TQMA8XXS
+
+config SYS_BOARD
+	default "tqma8xxs"
+
+config SYS_VENDOR
+	default "tqc"
+
+config SYS_CONFIG_NAME
+	default "tqma8xxs"
+
+choice
+	prompt "TQMa8XxS CPU variant"
+	default TQMA8XXS_CPU_MX8QXP
+	help
+	  Select cpu variant for TQMa8QXS
+
+config TQMA8XXS_CPU_MX8QXP
+	bool "TQMa8XQPS with i.MX8QXP"
+	help
+	  Select the CPU variant
+
+config TQMA8XXS_CPU_MX8DX
+	bool "TQMa8XDS with i.MX8DX"
+	help
+	  Select the CPU variant
+
+endchoice
+
+choice
+	prompt "TQMa8XxS RAM size"
+	default TQMA8XXS_RAM_1014MB if TQMA8XXS_CPU_MX8QXP
+	default TQMA8XXS_RAM_512MB if TQMA8XXS_CPU_MX8DX
+	help
+	  Select RAM size for TQMa8XxS
+
+config TQMA8XXS_RAM_512MB
+	bool "TQMa8XxS with 512 MiB RAM"
+	help
+	  Select variant with 512 MiB RAM
+
+config TQMA8XXS_RAM_1014MB
+	bool "TQMa8XxS with 1024 MiB RAM"
+	help
+	  Select variant with 1024 MiB RAM
+
+config TQMA8XXS_RAM_2048MB
+	bool "TQMa8XxS with 2048 MiB RAM"
+	help
+	  Select variant with 2048 MiB RAM
+
+endchoice
+
+choice
+	prompt "TQMa8XxS base board variant"
+	default TQMA8XXS_BB_MB_SMARC_2
+	help
+	  Select base board for TQMa8XxS
+
+config TQMA8XXS_BB_MB_SMARC_2
+	bool "TQMa8XxS on MB_SMARC_2 Board"
+	select TQC_SDMMC
+	help
+	  Select the MB_SMARC_2 board. This is a SMARC2 mainboard
+	  etc.
+
+endchoice
+
+config DEFAULT_DEVICE_TREE
+	default "fsl-imx8dx-tqma8xds-mb-smarc-2" if TQMA8XXS_CPU_MX8DX && TQMA8XXS_BB_MB_SMARC_2
+	default "fsl-imx8qxp-tqma8xqps-mb-smarc-2" if TQMA8XXS_CPU_MX8QXP && TQMA8XXS_BB_MB_SMARC_2
+
+config DEFAULT_FDT_FILE
+	default "fsl-imx8dx-tqma8xds-mb-smarc-2.dtb" if TQMA8XXS_CPU_MX8DX && TQMA8XXS_BB_MB_SMARC_2
+	default "fsl-imx8qxp-tqma8xqps-mb-smarc-2.dtb" if TQMA8XXS_CPU_MX8QXP && TQMA8XXS_BB_MB_SMARC_2
+
+choice
+	prompt "TQMa8XxS boot device"
+	default TQMA8XXS_BOOT_SDCARD
+	help
+	  Select boot device for TQMa8QXS
+
+config TQMA8XXS_BOOT_SDCARD
+	bool "TQMa8XxS from SD / e-MMC"
+	select SD_BOOT
+	select ENV_IS_IN_MMC
+	help
+	  Select booting from SD / e-MMC
+
+endchoice
+
+
+source "board/tqc/common/Kconfig"
+
+endif
diff --git a/board/tqc/tqma8xxs/Makefile b/board/tqc/tqma8xxs/Makefile
new file mode 100644
index 0000000000..1968b52fa4
--- /dev/null
+++ b/board/tqc/tqma8xxs/Makefile
@@ -0,0 +1,9 @@
+#
+# Copyright 2018 - 2019 TQ Systems GmbH
+#
+# SPDX-License-Identifier:	GPL-2.0+
+#
+
+obj-y += tqma8xxs.o
+obj-$(CONFIG_TQMA8XXS_BB_MB_SMARC_2) += tqma8xxs-mb-smarc-2.o
+
diff --git a/board/tqc/tqma8qxs/tqma8qxs-mb-smarc-2.c b/board/tqc/tqma8xxs/tqma8xxs-mb-smarc-2.c
similarity index 100%
rename from board/tqc/tqma8qxs/tqma8qxs-mb-smarc-2.c
rename to board/tqc/tqma8xxs/tqma8xxs-mb-smarc-2.c
diff --git a/board/tqc/tqma8qxs/tqma8qxs.c b/board/tqc/tqma8xxs/tqma8xxs.c
similarity index 100%
rename from board/tqc/tqma8qxs/tqma8qxs.c
rename to board/tqc/tqma8xxs/tqma8xxs.c
diff --git a/configs/tqma8dxs_mb_smarc_2_mmc_defconfig b/configs/tqma8xds_mb_smarc_2_mmc_defconfig
similarity index 96%
rename from configs/tqma8dxs_mb_smarc_2_mmc_defconfig
rename to configs/tqma8xds_mb_smarc_2_mmc_defconfig
index bfb1aa13fd..44dc5e1246 100644
--- a/configs/tqma8dxs_mb_smarc_2_mmc_defconfig
+++ b/configs/tqma8xds_mb_smarc_2_mmc_defconfig
@@ -3,8 +3,8 @@ CONFIG_ARCH_IMX8=y
 CONFIG_SYS_TEXT_BASE=0x80020000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_SMC_FUSE=y
-CONFIG_TARGET_TQMA8QXS=y
-CONFIG_TQMA8QXS_CPU_MX8DX=y
+CONFIG_TARGET_TQMA8XXS=y
+CONFIG_TQMA8XXS_CPU_MX8DX=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTDELAY=3
diff --git a/configs/tqma8qxs_mb_smarc_2_mmc_defconfig b/configs/tqma8xqps_mb_smarc_2_mmc_defconfig
similarity index 98%
rename from configs/tqma8qxs_mb_smarc_2_mmc_defconfig
rename to configs/tqma8xqps_mb_smarc_2_mmc_defconfig
index 43d9258bf7..72d15ea536 100644
--- a/configs/tqma8qxs_mb_smarc_2_mmc_defconfig
+++ b/configs/tqma8xqps_mb_smarc_2_mmc_defconfig
@@ -3,7 +3,7 @@ CONFIG_ARCH_IMX8=y
 CONFIG_SYS_TEXT_BASE=0x80020000
 CONFIG_SYS_MALLOC_F_LEN=0x2000
 CONFIG_SMC_FUSE=y
-CONFIG_TARGET_TQMA8QXS=y
+CONFIG_TARGET_TQMA8XXS=y
 CONFIG_IMX_BOOTAUX=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTDELAY=3
diff --git a/include/configs/tqma8qxs-mb-smarc-2.h b/include/configs/tqma8xxs-mb-smarc-2.h
similarity index 76%
rename from include/configs/tqma8qxs-mb-smarc-2.h
rename to include/configs/tqma8xxs-mb-smarc-2.h
index be2b231e11..7a66973665 100644
--- a/include/configs/tqma8qxs-mb-smarc-2.h
+++ b/include/configs/tqma8xxs-mb-smarc-2.h
@@ -1,11 +1,11 @@
 /*
- * Copyright 2018 TQ Systems GmbH
+ * Copyright 2018 - 2019 TQ Systems GmbH
  *
  * SPDX-License-Identifier:	GPL-2.0+
  */
 
-#if !defined(__TQMA8QXS_MB_SMARC_2_H)
-#define __TQMA8QXS_MB_SMARC_2_H
+#if !defined(__TQMA8XXS_MB_SMARC_2_H)
+#define __TQMA8XXS_MB_SMARC_2_H
 
 /* TODO: move to module ... */
 
@@ -30,4 +30,4 @@
 #define BB_ENV_SETTINGS \
 	"fdt_file=" CONFIG_DEFAULT_FDT_FILE "\0"
 
-#endif /* __TQMA8QXS_MB_SMARC_2_H */
+#endif /* __TQMA8XXS_MB_SMARC_2_H */
diff --git a/include/configs/tqma8qxs.h b/include/configs/tqma8xxs.h
similarity index 97%
rename from include/configs/tqma8qxs.h
rename to include/configs/tqma8xxs.h
index 74a5ede6c6..f65ad9260e 100644
--- a/include/configs/tqma8qxs.h
+++ b/include/configs/tqma8xxs.h
@@ -1,11 +1,11 @@
 /*
- * Copyright 2018 TQ Systems GmbH
+ * Copyright 2018 - 2019 TQ Systems GmbH
  *
  * SPDX-License-Identifier:	GPL-2.0+
  */
 
-#ifndef __TQMA8QXS_H
-#define __TQMA8QXS_H
+#ifndef __TQMA8XXS_H
+#define __TQMA8XXS_H
 
 #include <linux/sizes.h>
 #include <asm/arch/imx-regs.h>
@@ -304,11 +304,11 @@
 #define CONFIG_NR_DRAM_BANKS		1
 #define PHYS_SDRAM_1			0x80000000
 
-#if defined(CONFIG_TQMA8QXS_RAM_512MB)
+#if defined(CONFIG_TQMA8XXS_RAM_512MB)
 #define PHYS_SDRAM_1_SIZE		SZ_512M
-#elif defined(CONFIG_TQMA8QXS_RAM_1014MB)
+#elif defined(CONFIG_TQMA8XXS_RAM_1014MB)
 #define PHYS_SDRAM_1_SIZE		SZ_1G
-#elif defined(CONFIG_TQMA8QXS_RAM_2048MB)
+#elif defined(CONFIG_TQMA8XXS_RAM_2048MB)
 #define PHYS_SDRAM_1_SIZE		SZ_2G
 #else
 #error
@@ -365,18 +365,18 @@
 #define CONFIG_SYS_MEMTEST_SCRATCH CONFIG_SYS_MEMTEST_END
 #endif
 
-#if defined(CONFIG_TQMA8QXS_CPU_MX8QXP)
+#if defined(CONFIG_TQMA8XXS_CPU_MX8QXP)
 #define TQMA8_BOARD_NAME	"TQMa8XQPS"
 #define TQMA8_BOARD_REV		"iMX8XQPS"
-#elif defined(CONFIG_TQMA8QXS_CPU_MX8DX)
+#elif defined(CONFIG_TQMA8XXS_CPU_MX8DX)
 #define TQMA8_BOARD_NAME	"TQMa8XDS"
 #define TQMA8_BOARD_REV		"iMX8XDS"
 #else
 #error
 #endif
 
-#if defined(CONFIG_TQMA8QXS_MB_SMARC_2)
-#include "tqma8qxs-mb-smarc-2.h"
+#if defined(CONFIG_TQMA8XXS_BB_MB_SMARC_2)
+#include "tqma8xxs-mb-smarc-2.h"
 #else
 #error
 #endif
@@ -392,4 +392,4 @@
 #include <config_distro_bootcmd.h>
 #endif
 
-#endif /* __TQMA8QXS_H */
+#endif /* __TQMA8XXS_H */
